AndelaOSP/shufflebox-backend
# [Fix #138140325] return serialized user objects instead of primary keys

#### What does this PR do?
Fixes returning user ids instead of user objects
#### How to manually test this?
Start the server and access `http://localhost:8000/api/hangout/` on your browser. You should see something like this
```
    {
        "id": 1,
        "date": "2017-02-15",
        "members": [
            {
                "id": 1,
                "username": "jee",
                "email": "jeegiks@gmail.com",
                "profile": {
                    "avatar": "",
                    "birth_date": null,
                    "bio": ""
                }
            },
            {
                "id": 2,
                "username": "amos",
                "email": "amos@gmail.com",
                "profile": {
                    "avatar": "",
                    "birth_date": null,
                    "bio": ""
                }
            }
        ]
    }
```
#### Relevant pivotal tracker stories
138140325
